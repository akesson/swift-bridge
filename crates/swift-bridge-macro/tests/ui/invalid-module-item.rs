//! # To Run
//! cargo test -p swift-bridge-macro -- ui trybuild=invalid-module-item.rs

#[swift_bridge::bridge]
mod ffi {
    use std;
    fn foo() {}
}

fn main() {}
