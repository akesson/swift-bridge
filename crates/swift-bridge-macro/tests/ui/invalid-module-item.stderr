error: Only `extern` blocks, structs and enums are supported.
 --> tests/ui/invalid-module-item.rs:6:5
  |
6 |     use std;
  |     ^^^^^^^^

error: Only `extern` blocks, structs and enums are supported.
 --> tests/ui/invalid-module-item.rs:7:5
  |
7 |     fn foo() {}
  |     ^^^^^^^^^^^
